tools/internal_ci/linux/grpc_xds_k8s_xlang.sh

```diff
@@ -22,7 +22,12 @@ readonly TEST_DRIVER_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/${TES
 readonly IMAGE_REPO="gcr.io/grpc-testing/xds-interop"
 readonly SERVER_LANG="cpp go java"
 readonly CLIENT_LANG="cpp go java"
-readonly VERSION_TAG="master v1.50.x"
+readonly MASTER_BRANCH="${MASTER_BRANCH:-master}"
+readonly LATEST_BRANCH_FROM_GITHUB="$((git ls-remote -t --refs https://github.com/grpc/${GITHUB_REPOSITORY_NAME}.git | cut -f 2 | sed s#refs/tags/##) | sort -V | tail -n 1 | cut -f 1-2 -d.)".x
```

Comment:
Context: talked about it with @ejona86

Reply:
As discussed will use the version.sh file to determine the latest version. Also we decided we will test against the branch (e.g. 1.50.x) instead of a tag (e.g. 1.50.1)

Reply:
done